### PR TITLE
feat: upgrade skipper to v0.10.200 (#1965)

### LINKED ISSUE #1965: Upgrade Skipper to v0.10.200
To roll out:

* default filter
* LB algorithms
etc.

### PR DESCRIPTION
**Pull request by tkrop**
fixes #1965.

Signed-off-by: tkrop <tronje.krop@zalando.de>

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

File: cluster/manifests/skipper/deployment.yaml [hunk 1]
```diff
@@ -80,6 +80,7 @@ spec:
           - "-api-usage-monitoring-realm-keys=https://identity.zalando.com/realm"
           - "-api-usage-monitoring-client-keys=https://identity.zalando.com/managed-id,sub"
           - "-api-usage-monitoring-default-client-tracking-pattern=services[.].*"
+          - "-default-filters-dir=/etc/config/default-filters"
```
Comment (mikkeloscar) [inline cluster/manifests/skipper/deployment.yaml:83]: Will this work if the file doesn't exist?

Comment (szuecs) [reply cluster/manifests/skipper/deployment.yaml:83]: It's a directory and it is in the volume and yes it is checked before they open files.

Comment (mikkeloscar) [reply cluster/manifests/skipper/deployment.yaml:83]: I see it's behind the if guard, missed it in the diff view :)

**Comment by szuecs** [comment]
cadvisor did not became ready

**Comment by szuecs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**